src/defaults.js

```diff
@@ -85,5 +85,11 @@ export default {
      * window instance
      * @window {object}
      */
-    window: window
+    window: window,
+
+    /**
+     * If false, slides lory to the first slide on window resize.
+     * @persistIndex {boolean}
+     */
+    persistIndex: true
```

Comment:
Sorry for the late answer: I would say to not have breaking changes, default should be false. And I would name this `rewindOnResize` to be consistent